integration_tests/run_integration_tests.sh

```diff
@@ -25,13 +25,13 @@ fi
 
 pushd "$(dirname "${BASH_SOURCE[0]}")"
 
-python3 -m  venv venv
+python3 -m venv venv
 
 source venv/bin/activate
 
 python3 -m pip install -r requirements.txt
 
-pytest
+pytest -v
```

Comment:
Calling this out as I just want to be extra sure that adding more verbose output won't result in any chance of any credentials or sensitive info being dumped out to logs or anything, even if this is just test data.

Reply:
I don't think it will, but if it does:
- it will only be to concourse, which we consider secure
- the tests already do this, since we `set -x` in the script that runs the test